.github/workflows/ci.yml

```diff
@@ -65,3 +72,21 @@ jobs:
 
     - name: Docs
       run: cargo doc --features docs,unstable
+
+  clippy_check:
```

Comment:
It is cumbersome to set triggers in multiple places, so merge clippy.yml into ci.yml.